uploader.json

```diff
@@ -1500,6 +1505,11 @@
             "pattern": [],
             "symbolic_name": "httpd_conf_scl_jbcs_httpd24"
         },
+        {
+            "file": "/opt/rh/jbcs-httpd24/root/etc/httpd/conf.modules.d/()*.+\\.conf",
+            "pattern": [],
+            "symbolic_name": "httpd_modules_conf_scl_jbcs_httpd24"
```

Comment:
@huali027 the symbolic name needs to match the spec name that you used in insights-core here, and on line 1496, it should not be a new name.

Reply:
@bfahr, sure, I see, I'll update it, thanks.